src/compiler/diagnosticMessages.json

```diff
@@ -1943,6 +1943,10 @@
         "category": "Error",
         "code": 2689
     },
+    "Module name should not include a '.ts' extension: '{0}'.": {

```

Comment:
`An import path should not end with a '{0}' extension. Consider importing '{1}' instead.`

Since this error is being used for `.tsx` as well.